### ISSUE TITLE
"Launch" button is confusing

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#52561]
Upon signing up for a new site, I'm dropped into the editor. I dismissed the welcome tour because I would do that normally, so I don't have much context about how my site exists in the system yet.

I'm presented with these actions at the top right:

<img width="365" alt="Screen Shot 2021-05-04 at 10 10 10 AM" src="https://user-images.githubusercontent.com/2124984/117017404-91d12300-acc1-11eb-82da-533f7bc095ea.png">

Preview is pretty self-explanatory, and does what I expect it to do. ✅ 

Update is not so self-explanatory, but it's not the primary action, so maybe I don't need to worry about it yet.

Next is "Launch". But "Launch" what? Would it be more clear to say "Launch site"? (Did it say something like that before?)

"Switch to draft" is equally confusing because I don't yet know what a "draft" is, necessarily. "Switch to draft" implies this site is already published, but the button says Launch, which implies it's not...

Basically I have no context for what state my site is in, based on the actions presented to me. Also, having four actions in a row is super confusing/overwhelming. 😬

**Comment by sixhours** [comment]
Closing in favor of #52590
